### DISCUSSION TITLE
Provide ability to control when "tasks done" is rendered for checkboxes

### DISCUSSION THREAD

**Discussion by wsmelton** [community/community#4598] [Projects and Issues]
The issue forms templates provide the ability to have checkboxes to collect information. The same holds true for markdown templates. Repositories do not all use checkboxes to reference tasks that need to be completed. 

Please provide an option whether it be from markdown or an attribute in the issues forms to have valid checkboxes used without them causing the "tasks done" every time.

GitHub's own example for issue forms shows a checkbox for "which browser is being used". [See here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms). Using that in an issue will cause the "Tasks done" to be rendered below the Issue title.

We should be able to control this being rendered on an issue:

![image](https://user-images.githubusercontent.com/11204251/125180255-992f1400-e1bd-11eb-9992-e367ada30ab0.png)

**Comment by GiovanH** [comment]
Having this same issue in a case where checkboxes do not necessarily correspond to tasks, but to reporting user configuration. Automatic task lists have rendered a large backlog of issues incorrect, due to their assumption that all checkboxes are todos. There needs to be a specifier for this behavior.
